### ISSUE TITLE
[Crash] LotR Mod - java.lang.IndexOutOfBoundsException: Index: 37, Size: 35

### ISSUE THREAD

**Issue by juniorjunior** [CyberdyneCC/Thermos#133]
I am having a problem with The Lord of the Rings mod ([from here](http://lotrminecraftmod.wikia.com)).

Crash Log: http://pastebin.com/Lf97emPi
Crash Log (possibly related): http://pastebin.com/ypb9WRSV
Crash Log (possibly related): http://pastebin.com/GjNR6qmQ

Neither of these crashes seem to happen when using Forge as the server process. When working in previously generated chunks there seems to be no problem, and logging back in after a crash produced by moving/worldgen seems to work fine (i.e. no world corruption). The crash seems to occur when moving through the world as new chunks are being generated. Travel speed (mounted/flying/walking) does not seem to effect the issue.

**Comment by linkinkov** [comment]
This mod never work with cauldron, kcauldron, Thermos :)

**Comment by juniorjunior** [comment]
Yeah, I know they don't go to any lengths to support it, but I've used it in the past with no troubles (v22.3 used extensively in the past with Cauldron).

**Comment by Robotia** [comment]
If it has work in past, I will definitely support now

**Comment by Robotia** [comment]
@juniorjunior I will have compat ready soon

**Comment by juniorjunior** [comment]
New crash report using Build-30: http://pastebin.com/KSGDpe5v

**Comment by Robotia** [comment]
Hmm ok

**Comment by Robotia** [comment]
@juniorjunior try disabling spigot async lighting

**Comment by juniorjunior** [comment]
So, I cloned off the main server to my dev server, switched to using Thermos Build #30 (same as before for me) and traveled for many thousands of blocks. I left that option enabled to get a baseline, but I didn't get a single crash. Lack of evidence proves little, but I don't know what else to do other than a ton of chunk-gen. For the main server I have switched from Thermos to Forge (adding ForgeEssentials and CoFHTweaks) and haven't had a crash in over a week. I do appreciate the effort put in, but I don't anticipate switching back at the moment.
